packages/ember-auto-import/ts/package.ts

```diff
@@ -250,13 +250,19 @@ export default class Package {
 
   // the semver range of the given package that our package requests in
   // package.json
-  requestedRange(packageName: string): string | undefined {
+  requestedRange(
+    packageName: string,
+    includeDevDeps: boolean
```

Comment:
This shouldn't be an argument. `includeDevDeps === !this.isAddon`

Reply:
🙈 I really should have noticed I was inside the package here 😫

Reply:
fixed 👍